de.uni_heidelberg.zah.GaiaSky.yaml

```diff
@@ -0,0 +1,44 @@
+app-id: de.uni_heidelberg.zah.GaiaSky
+runtime: org.freedesktop.Platform
+runtime-version: '19.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions: [org.freedesktop.Sdk.Extension.openjdk11]
+command: gaiasky
+finish-args:
+  - --socket=x11
+  - --socket=wayland
+  - --device=dri
+  - --socket=pulseaudio
+  - --share=network
+  - --env=PATH=/app/bin:/app/jre/bin:/usr/bin
+  - --env=JAVA_HOME=/app/jre
+modules:
+  - name: openjdk
+    buildsystem: simple
+    build-commands: [/usr/lib/sdk/openjdk11/install.sh]
+
+  - name: gaiasky
+    buildsystem: simple
+    build-commands:
+       - mkdir -p /app/bin/bookmarks/ /app/bin/conf/ /app/bin/i18n/ /app/bin/lib/ /app/bin/mappings/ /app/bin/music/ /app/bin/scripts/
```

Comment:
Nitpick: Any reason for not using install here as well?

Reply:
By my reading install can't copy entire directories.. maybe I got it wrong?

Reply:
Never mind